src/NGAC/ngac_types/resource.py

```diff
@@ -0,0 +1,53 @@
+"""
+Resource abstraction
+
+A resource can be a file or something else
+"""
+
+
+def relative_import(package, module):
+    """
+    Solves pythons shitty relative import problem
+    """
+    import os
+
+    exec(
+        f"""from {"" if os.getcwd().endswith("ngac_types") else "ngac_types." if os.getcwd().endswith("NGAC") else "NGAC.ngac_types."}{package} import {module}"""
+    )
+    return locals()[module]
+
+
+NgacObject = relative_import("ngac_object", "NgacObject")
+OA = relative_import("ngac_attribute", "ObjectAttribute")

```

Comment:
About the names. Would be not be better to give the variables a little longer names? For example I think that OA makes sense but could take a little time to understand when having a lot of things in their mind. So maybe obj_attr? or something still short but so it is not confused as a temporary name. Especially since it only makes sense if a person has a little pre-knowledge of the content. It will also be easier when it's an argument for a function, thus it shows the full name of the variables.

Some of the other names is res - in resources.py, pol - in policy.py - UA - for user attributes

Reply:
Since it's an internal library and will not undergo large changes after it's completion, I think that we can justify longer variable names, so I don't think that we need to shorten it at all,
```python
ObjectAttribute = relative_import("ngac_attribute","ObjectAttribute")
```
So there can be no missunderstanding

Reply:
I think that the new naming is better, look at it again when you have time @Leohemmingsson

Reply:
Looks good!